components.js

```diff
@@ -341,6 +350,10 @@ var components = {
 			"require": "css",
 			"owner": "Golmote"
 		},
+		"lisp": {
+			"title": "Lisp",
+			"owner": "JuanCaicedo"
+		},
```

Comment:
You added `elisp` and `lisp` in this file, but not `emacs`.
The general rule here is to only add one entry, the _main_ one. So I guess here it should be `lisp` since it's the most generic. Please remove the `elisp` entry.

That also means the component should be renamed to `prism-lisp`, along with the test folder and so on.

Reply:
You could add alias for emacs and elisp